cli/gdb.ts

```diff
@@ -63,11 +77,11 @@ function getOpenOcdPath() {
     let cmd = `log_output built/openocd.log; ${script}; init; halt;`
 
     let args = [openocdBin, "-d2",
-        "-s", openocdPath + "/share/openocd/scripts/",
+        "-s", openocdPath + "share/openocd/scripts/",
```

Comment:
the ``/`` is removed, will this work in Arduino scenario? Alternatively, use path.join(...) to avoid path issues.